z3alpha/multiprocess_evaluator.py

```diff
@@ -16,21 +16,21 @@
 
 def run_solver(
     solver_path, smt_file, timeout, id, strategy=None, tmp_dir="/tmp/", 
-    cpu_limit=1, memory_limit=None, monitor_resources=False
+    cpu_limit=1, memory_limit=None, monitor_resources=False, quiet=False
```

Comment:
May be set a log level as argument, and use log.debug instead of this quiet argument?